Lib/test/test_random.py

```diff
@@ -42,10 +42,16 @@ class MySeed(object):
             def __hash__(self):
                 return -1729
         for arg in [None, 0, 0, 1, 1, -1, -1, 10**20, -(10**20),
```

Comment:
```suggestion
        for arg in [None, 0, 1, -1, 10**20, -(10**20),
```
I suppose they were `int` and `long` literals in Python 2.